cli/Valet/Drivers/Specific/BedrockValetDriver.php

```diff
@@ -11,10 +11,11 @@ class BedrockValetDriver extends BasicValetDriver
      */
     public function serves(string $sitePath, string $siteName, string $uri): bool
     {
-        return file_exists($sitePath.'/web/app/mu-plugins/bedrock-autoloader.php') ||
-              (is_dir($sitePath.'/web/app/') &&
-               file_exists($sitePath.'/web/wp-config.php') &&
-               file_exists($sitePath.'/config/application.php'));
+        return $this->composerRequires('roots/bedrock-autoloader') ||
```

Comment:
https://github.com/laravel/valet/blob/50a437e679867ecc9da90e8e1b740dcdd0bab553/cli/Valet/Drivers/ValetDriver.php#L204

`composerRequires` needs you to pass the `$sitePath` in as the first parameter

Reply:
Fixed and actually ran tests locally.

Sorry about the messy commit w/ indentation. There didn't seem to be a convention I was familiar with so I tried to make it 4-space indents consistently where I touched things.